packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -184,7 +224,7 @@ const main = () => {
                 'a message to include along with the uploaded bundle in Managed Runtime'
             )
                 // The default message is loaded dynamically as part of `uploadBundle(...)`
-                .default(undefined, '<git branch>:<git commit hash>')
+                .default(null, '<git branch>:<git commit hash>')
```

Comment:
I changed this to null because I noticed that when it's undefined, the default message is not shown in the help output because of [this](https://github.com/tj/commander.js/blob/master/lib/help.js#L245).